### DISCUSSION TITLE
Downloading Node with custom root

### DISCUSSION THREAD

**Discussion by RiverOmega** [quarkiverse/quarkus-quinoa#646] [Q&A] [answered]
Hello! 

The property `quarkus.quinoa.package-manager-install.node-download-root` allows me to download from a custom registry, which is nice. But Quinoa itself concatenates the root with `/{node-version}/node-{node-version}-linux-x64.tar.gz`, and I can't seem to find a way to change this, because that is not how things are structured in the registry I'm working with.

Is there any workaround to this? In my case I have:

`root/node-{node-version}.tgz`

**Comment by melloware** [comment]
@RiverOmega good point i will create an issue from this right now.

**Reply by RiverOmega** [reply, marked as answer]
Perfect, I'll close this discussion, as issue [647](https://github.com/quarkiverse/quarkus-quinoa/issues/647) was created.
